core-kotlin-modules/core-kotlin-datastructures/src/test/kotlin/com/baeldung/binarydecimal/BinaryDecimalConversionTest.kt

```diff
@@ -0,0 +1,68 @@
+package com.baeldung.binarydecimal
+
+import org.junit.jupiter.api.Test
+import kotlin.math.pow
+import kotlin.test.assertEquals
+
+class BinaryDecimalConversionTest {
+
+    @Test
+    fun `binary to decimal using toInt() method`(){
+        assertEquals(439, "110110111".toInt(2))
+        assertEquals(109, "1101101".toInt(2))
+        assertEquals(27, "11011".toInt(2))
+        assertEquals(122979, "11110000001100011".toInt(2))
```

Comment:
You have some prime and non-prime numbers, which is good, but all are odd. For this and the other tests, let's convert at least one even number to show our solutions work for those as well.